addons/web/static/src/views/list/list_controller.js

```diff
@@ -54,7 +54,7 @@ export class ListController extends Component {
         this.rootRef = useRef("root");
 
         this.archInfo = this.props.archInfo;
-        this.editable = this.props.editable ? this.archInfo.editable : false;
+        this.editableDirection = this.props.readonly ? this.archInfo.editable : false;
```

Comment:
I would keep the former terminology, because we're used to it (and `direction` is not really accurate, because it can be false as well, which in this case indicates that it's *not* editable)

Reply:
also, missing negation. This could explain why runbot is red :D